src/parameters/query/scripts/IncludeInActive.yaml

```diff
@@ -0,0 +1,7 @@
+in: query
+name: includeInActive
+required: false
+schema:
+  type: string
+description: allow for filtering InActive Script. By default it will return only Active Scripts. Pass includeInActive=true to get Active and Inactive Script.
```

Comment:
remove casing e.g. InActive Script -> inactive script.
This does not allow for filtering, it allows for including.
Can't you define a set of allowed values (enum) in OpenApi.